flexmeasures/ui/templates/base.html

```diff
@@ -85,20 +85,23 @@
                             <span class="caret"></span></a>
                         <ul class="dropdown-menu">
                             <li {% if current_user.has_role('anonymous') %}class="disabled" {% endif %}><a
-                                    {% if not current_user.has_role('anonymous') %}href="/tasks/forecasting"
+                                    {% if not current_user.has_role('anonymous') %}href="/tasks/0/view/jobs/forecasting/started/10/1"
                                     {% endif %}>Forecasts</a></li>
                             <li {% if current_user.has_role('anonymous') %}class="disabled" {% endif %}><a
-                                    {% if not current_user.has_role('anonymous') %}href="/tasks/scheduling"
+                                    {% if not current_user.has_role('anonymous') %}href="/tasks/0/view/jobs/scheduling/started/10/1"
                                     {% endif %}>Schedules</a></li>
                             <li {% if current_user.has_role('anonymous') %}class="disabled" {% endif %}><a
-                                    {% if not current_user.has_role('anonymous') %}href="/tasks/labelling"
+                                    {% if not current_user.has_role('anonymous') %}href="/tasks/0/view/jobs/reporting/started/10/1"
+                                    {% endif %}>Reports</a></li>
+                            <li {% if current_user.has_role('anonymous') %}class="disabled" {% endif %}><a
+                                    {% if not current_user.has_role('anonymous') %}href="/tasks/0/view/jobs/labelling/started/10/1"
```

Comment:
Queues like labelling and alreting are not even planned at this point. I believe they should not be in the menu.

Reply:
I agree. I'll generate the list of queues from whatever is registered in `app.queues`, which would then also put us on the path of creating a clean way for plugins to register their own queues (which is not currently needed).